src/main/java/codeu/model/data/Media.java

```diff
@@ -53,7 +56,19 @@ public Instant getCreationTime() {
   }
 
   /** Returns the content of this Media. */
-  public byte[] getContent() {
+  public BufferedImage getContent() {
```

Comment:
Rename 'content' to 'bufferedImage' or just 'image'?
Rename this method to getBufferedImage or getImage?

The reason being, this is a Media.java file, technically it might handle other non-image content.